SD-K9/src/app/components/map-box/map-box.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { IonicModule } from '@ionic/angular';
+
+import { MapBoxComponent } from './map-box.component';
+
+// describe('MapBoxComponent', () => {
```

Comment:
Do your changes affect the tests? Is it necessary to create a separate issue to refactor the tests?

Reply:
As I replied to Paul, the goal of this pr is to provide the new file structure and factory design, so issues that are blocked by this pr can be unblocked.
Since I modified a lot of files and the inputs of the floor-plan.component, many tests will fail. Those I commented out are the ones that will fail. It is easier to update these tests once the  factory pattern is combined with the routing pattern.
The update of these tests are associated to a task within the same issue as US-48, it is not a separate issue.

Reply:
As long as this is documented, we will merge to unblock the issues that are being blocked